systems/plants/@RigidBodyManipulator/contactConstraints.m

```diff
@@ -57,7 +57,8 @@
 % For now, all coefficients of friction are 1
 mu = ones(nC,1);
 
-d = obj.surfaceTangents(normal);
+d = surfaceTangentsmex(normal);
```

Comment:
we check for the existence of the mex file in some places, but it's not clear that you'd want to do that here (for efficiency's sake).  seems like we need to check for it somewhere slightly upstream.

On Jan 31, 2015, at 4:46 PM, John Carter <notifications@github.com> wrote:

> In systems/plants/@RigidBodyManipulator/contactConstraints.m:
> 
> > @@ -57,7 +57,8 @@
> >  % For now, all coefficients of friction are 1
> >  mu = ones(nC,1);
> >  
> > -d = obj.surfaceTangents(normal);
> > +d = surfaceTangentsmex(normal);
> You're right.
> 
> Is there a standard way to check for the presence of Eigen from MATLAB? Would the existence of the compiled mex file be sufficient?
> 
> —
> Reply to this email directly or view it on GitHub.
>